app/components/container-subpod/template.hbs

```diff
@@ -1,10 +1,17 @@
-{{action-menu model=model showPrimary=false}}
 
 {{#if groupHasChildren}}
 <div class="subpod-children clearfix">
   <h6>Sidekicks</h6>
   {{#each children as |child|}}
-    {{container-dot model=child}}
```

Comment:
Can the tooltip-element be put into the container-dot component instead of duplicating this several places?

You'll have to remove the custom render() function, but I think that's deprecated anyway..